docs/cosmology/index.rst

```diff
@@ -299,18 +298,20 @@ should consult the cosmology module source code for details.
 
 Photons and Neutrinos
 ---------------------
-The cosmology classes include the contribution to the energy density
+The cosmology classes (can) include the contribution to the energy density
 from both photons and neutrinos.  By default, the latter are assumed
 massless.  The three parameters controlling the properties of these
 species, which are arguments to the initializers of all the
 cosmological classes, are ``Tcmb0`` (the temperature of the CMB at z=0),
 ``Neff``, the effective number of neutrino species, and ``m_nu``, the rest
 mass of the neutrino species.  ``Tcmb0`` and ``m_nu`` should be expressed
-as unit Quantities.  All three have standard default values (2.725 K,
+as unit Quantities.  All three have standard default values (0 K,
 3.04, and 0 eV respectively; the reason that ``Neff`` is not 3 primarily
 has to do with a small bump in the neutrino energy spectrum due to
 electron-positron annihilation, but is also affected by weak
-interaction physics).
+interaction physics).  Setting the CMB temperature to zero removes
+the contribution of both neutrinos and photons.  This is the default
+so that users do not include components by default that they did not request.
 
```

Comment:
Maybe replace the final sentence with

"This is the default to ensure these components are excluded unless the user explicitly requests them."